docs/faststore/docs/go-live/2-configuring-external-dns.mdx

```diff
@@ -41,13 +41,18 @@ Now, you must set up your VTEX account to use the DNS records created in the pre
 1. On VTEX Admin, click your profile avatar, which can be identified by the first letter of your email address.
 2. Access **Account Settings > Account**.
 3. Go to the **Stores** tab and in the **Hosts** table, click the menu (**⋮**) and choose **Edit**.
-4. In the **Hosts** field, enter your main domain (e.g., `mystore.com`) to the list.
+4. In the **Hosts** field, enter your main domain (e.g., `www.mystore.com`) to the list.
 5. Click `Add`.
 6. Add the `secure` subdomain (e.g., `secure.mystore.com`) to the list.
    ![host-configuration](https://vtexhelp.vtexassets.com/assets/docs/src/account-settings-new___2e13bee3041e8505922a737b94652ca6.png)
-7. Click the **Save** button.
+7. Add the root domain as well; e.g., if your domain is `www.mysite.com,` add `mysite.com.` For a complete setup, you must add three hosts for each domain:
+   - `www.mysite.com`  (store domain)
+   - `mysite.com` (root domain)
+   - `secure.mysite.com` (checkout/myaccount subdomain)
```

Comment:
suggestion: since the previous examples are with `mystore` (for example, the one in the screenshot), what do you think about changing here to match that?
```suggestion
   - `www.mystore.com`  (store domain)
   - `mystore.com` (root domain)
   - `secure.mystore.com` (checkout/myaccount subdomain)
```